osu.Game/Online/API/Requests/Responses/CommentBundle.cs

```diff
@@ -47,17 +47,25 @@ public List<Comment> Comments
         [JsonProperty(@"included_comments")]
         public List<Comment> IncludedComments { get; set; }
 
+        private List<long> userVotes;
+
         [JsonProperty(@"user_votes")]
-        private List<long> userVotes
+        public List<long> UserVotes
         {
-            set => value.ForEach(v =>
+            get => userVotes;
+            set
             {
-                Comments.ForEach(c =>
+                userVotes = value;
+
+                value.ForEach(v =>
                 {
-                    if (v == c.Id)
-                        c.IsVoted = true;
+                    Comments.ForEach(c =>
+                    {
+                        if (v == c.Id)
+                            c.IsVoted = true;
+                    });
                 });
-            });
+            }
         }
```

Comment:
This setter has a potential bug. Let's say you do:

```csharp
bundle.UserVotes = new List<long> { 1, 2, 3 };
bundle.UserVotes = new List<long> { 4, 5 };
// comments w/ ID 1, 2, 3 will remain with IsVoted = true after set
```
which might not have been a problem when this was private, but is more of a problem now that you're exposing it publicly.